src/fastertransformer/models/llama/Llama.h

```diff
@@ -0,0 +1,221 @@
+/*
+ * Copyright (c) 2020-2023, NVIDIA CORPORATION.  All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#pragma once
+
+#include <cstddef>
+#include <vector>
+
+#include "src/fastertransformer/layers/DynamicDecodeLayer.h"
+#include "src/fastertransformer/models/llama/LlamaContextDecoder.h"
+#include "src/fastertransformer/models/llama/LlamaDecoder.h"
+#include "src/fastertransformer/models/llama/LlamaWeight.h"
+#include "src/fastertransformer/utils/custom_ar_comm.h"
+#include "src/fastertransformer/utils/prompt_learning.h"
+
+namespace fastertransformer {
+
+template<typename T>
+class Llama: public BaseLayer {
+private:
+    // meta data
+    size_t head_num_;
+    size_t size_per_head_;
+    size_t inter_size_;
+    size_t num_layer_;
+    size_t vocab_size_;
+    size_t rotary_embedding_dim_;
+
+    static constexpr bool  neox_rotary_style_ = true;
+    static constexpr float layernorm_eps_     = 1e-6f;
```

Comment:
This shouldn't be hardcoded to 1e-6 because the RMS layer norm epsilon is 1e-5 for the 65B model

Reply:
yep, changed to read from config

Reply:
There is no a default `layernorm_eps` in `llama_config.ini`, so run llama_example will failed, Do you need to configure a default value?

Reply:
I removed the default value on purpose, because a bad default value may cause a lot of time to debug the issue...

You could either set correct `layernorm_eps` value manually to `llama_config.ini` (7b: `1e-6f`, 65b: `1e-5f`), or reconvert the weights with `huggingface_llama_convert.py` @Lzhang-hub

Reply:
got it, thank you for your reply